Legion/VaultOfTheWardens/Tormentorum.lua

```diff
@@ -115,3 +123,63 @@ end
 function mod:FrighteningShout(args)
 	self:Message(args.spellId, "Urgent", "Warning", CL.casting:format(args.spellName))
 end
+
+do
+	local prev = 0
+	function mod:ShadowCrashDamage(args)
+		local t = GetTime()
+		if self:Me(args.destGUID) and t-prev > 1.5 then
+			prev = t
+			self:Message(args.spellId, "Personal", "Alert", CL.underyou:format(args.spellName))
+		end
+	end
+end
+
+do
+	local prev = 0
+
+	local function printStacks(self, key)
+		local meOnly = self:CheckOption(key, "ME_ONLY")
+		if meOnly then
+			local stacks = select(4, UnitDebuff("player", self:SpellName(key))) or 0

```

Comment:
Save the spellname in a variable in the do block to avoid this lookup every time.